### PR TITLE
Fix tests/deprecations and Symfony 5 support

### LINKED ISSUE #269: Symfony 5 support
Hello,

Do you plan to add Symfony 5 support? I forked this bundle and made it ready for Symfony 5 but there is a BC break that will make Symfony 3.4 no longer supported on PHP less than 7.4 : the `DataCollectorInterface::collect` signature changed and contravariant parameter support has only been added in PHP 7.4.

I can provide you the PR but what can we do about Symfony 3.4?

### PR DESCRIPTION
**Pull request by norkunas**
Closes #269

### CODE REVIEW

**Comment by norkunas** [comment]
@Nyholm do you think we could upgrade phpunit-bridge to v4.4 for this?

**Comment by Nyholm** [comment]
I like this PR. 

Yes. We don’t need to support different versions of the bridge nor does it matter what versions we use. 

We can use the version that is easiest for us. 
:)

**Comment by norkunas** [comment]
Also getting some deprecations from your symfony-test-bundle:
```
30x: Getting the container from a non-booted kernel is deprecated since Symfony 4.4.
    25x in ProviderFactoryTest::testProviderConfiguration from Bazinga\GeocoderBundle\Tests\Functional
    1x in BundleInitializationTest::testBundleWithOneProviderConfiguration from Bazinga\GeocoderBundle\Tests\Functional
    1x in BundleInitializationTest::testBundleWithCachedProvider from Bazinga\GeocoderBundle\Tests\Functional
    1x in BundleInitializationTest::testCacheLifetimeCanBeNull from Bazinga\GeocoderBundle\Tests\Functional
    1x in BundleInitializationTest::testBundleWithPluginsYml from Bazinga\GeocoderBundle\Tests\Functional
    1x in BundleInitializationTest::testBundleWithPluginXml from Bazinga\GeocoderBundle\Tests\Functional
```
:slightly_smiling_face:

**Comment by Baachi** [comment]
@norkunas The [symfony-bundle-test](https://github.com/SymfonyTest/symfony-bundle-test) is now compatible with symfony 5. So your tests now should pass on travis ci :)

**Comment by norkunas** [comment]
@Baachi great, thanks. :)

**Comment by norkunas** [comment]
Still getting the `Cannot retrieve the container from a non-booted kernel.`, while tests uses `$this->bootKernel()` from the `nyholm/symfony-test-bundle`.

**Comment by Baachi** [comment]
Will look into this today (if the hotel wlan let me) :)

**Comment by norkunas** [comment]
@Baachi did you find time to check this? :)

**Comment by Baachi** [comment]
I will check it today. My Hotel wlan wasn’t good enough sadly

File: Tests/Command/GeocodeCommandTest.php [hunk 1]
```diff
@@ -44,26 +46,24 @@ public function testExecute()
             'countryCode' => $country->getCode(),
         ]);
 
-        $geocoder = $this->getMockBuilder(ProviderAggregator::class)->getMock();
+        $geocoder = $this->createMock(ProviderAggregator::class);
```
Comment (Nyholm) [inline Tests/Command/GeocodeCommandTest.php]: Are you sure about this change?

Wasn't `createMock` deprecated it PHPUnit6?

Comment (norkunas) [reply Tests/Command/GeocodeCommandTest.php]: It does exist in 8.2

**Pull request merged**